fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -164,6 +164,7 @@
     public static final String RUNTIME_JOIN_FILTER_PUSH_DOWN_LIMIT = "runtime_join_filter_push_down_limit";
     public static final String ENABLE_GLOBAL_RUNTIME_FILTER = "enable_global_runtime_filter";
     public static final String ENABLE_COLUMN_EXPR_PREDICATE = "enable_column_expr_predicate";
+    public static final String ENABLE_FILTER_NULL_VALUES = "enable_filter_null_values";
```

Comment:
Add this session variable in order to test?

Reply:
In order to see performance comparison. Can be removed later.